.buildkite/notebook-parameters.yml

```diff
@@ -5,6 +5,8 @@ epochs: 2
 batch_size: 5
 # short random walks
 walk_length: 5
+# a small number of random walks
+walk_number: 2
```

Comment:
it might be best to move this to the bottom, with a comment about which notebooks use it, since it's not widespread.